cmd/util/cmd/execution-state-extract/cmd.go

```diff
@@ -55,6 +56,9 @@ func init() {
 
 	Cmd.Flags().BoolVar(&flagNoReport, "no-report", false,
 		"don't report the state")
+
+	Cmd.Flags().BoolVar(&flagSafeMode, "safe-mode", false,
```

Comment:
and as discussed this flag should be true by default in my view,

Reply:
Added in bcc34f2